styleguide/components/21-atoms/field-description/_field-description.scss

```diff
@@ -1,19 +1,21 @@
 .field-description {
-
-  @include theme('background-color', 'color-primary--lighten-1', 'field-description-background');
-  @include theme('color', 'color-primary', 'field-description-color');
+  @include theme('background-color', 'color-primary--lighten-4', 'field-description-background');
+  @include theme('color', 'color-tertiary--lighten-1', 'field-description-color');
+  @include icon('accolade-full', 'after');
```

Comment:
👎

Reply:
We'll need to discuss this one further next week.

Reply:
Decided to implement this through an SVG image for most instances. Pictures will need to be implemented with an inverted SVG.

Reply:
So we can remove the icon?

Reply:
I removed it for now, will implement the right one when I get it from one of the designers.